client/app/scripts/superdesk-publish/views/http-push-config.html

```diff
@@ -0,0 +1,8 @@
+<div class="field">
+    <label for="http_host" translate>Resource URL</label>
+    <input type="text" id="http_host" placeholder="{{ :: 'Resource URL'|translate }}" ng-model="destination.config.resource_url" required ng-change="$parent.setConfig(destination)">
+</div>
+<div class="field">
+    <label for="http_host" translate>Item URL</label>
+    <input type="text" id="http_host" placeholder="{{ :: 'Item URL'|translate }}" ng-model="destination.config.item_url" required ng-change="$parent.setConfig(destination)">
```

Comment:
Would it make sense to defer the model update by a few hundred milliseconds, i.e. when the user has likely finished with typing?